docs/source/core.rst

```diff
@@ -0,0 +1,26 @@
+Core Reference
+=============
```

Comment:
I've only included the modules I took the time to carefully docstring—don't think we should put the others in until they're cleaned up. Maybe we should add a "core reference in progress" qualifier somewhere?